entrypoints/content/pages/settings/AppearanceForm.tsx

```diff
@@ -0,0 +1,108 @@
+// Packages:
+import useUserPreferences from '../../hooks/useUserPreferences'
+
+// Typescript:
+import { Theme, Visibility } from 'types/user-preferences'
+
+// Components:
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '../../components/ui/select'
+
+// Functions:
+function AppearanceForm() {
+  // Constants:
+  const {
+    // TODO: Disable all inputs across /settings when isUserPreferencesLoading is true.
+    isLoading: isUserPreferencesLoading,
+    moderation,
+    setCheckOwnCommentForOffensiveSpeech,
+    setUnsafeContentPolicy,
+    setUnsafeWebsitePreviewsPolicy,
+  } = useUserPreferences()
+
+  // Return:
+  return (
+    <>
+      <div className="flex flex-row justify-between align-middle items-center">
+        <div className="flex flex-col">
+          <h3 className="text-sm font-base text-brand-primary">Theme</h3>
+          <p className="text-xs font-base text-brand-secondary">
+            Select the theme for the extension.
+          </p>
+        </div>
+        <Select defaultValue={Theme.System}>
+          <SelectTrigger className="text-xs w-44 h-fit px-2.5 py-2">
+            <SelectValue placeholder="Select Theme" />
+          </SelectTrigger>
+          <SelectContent>
+            <SelectGroup>
+              <SelectItem value="Light">Light</SelectItem>
+              <SelectItem value="Dark">Dark</SelectItem>
+              <SelectItem value="System">System Default</SelectItem>
```

Comment:
don't mix string and enum together, since we're using enums already, we'll have `value={Theme.Light}` and so on here.

for the onChange, we'll have `onChange={value => setTheme(value as Theme)}` (`setTheme` is just an example, the setter name may vary) - essentially, we are explicitly typecasting the string to `Theme` when setting it

Reply:
also btw there are examples in the other two forms as well on how the typecasting works, so you can look at that for reference

Reply:
@Sharan420 i pointed this out earlier also